SConstruct

```diff
@@ -748,3 +752,12 @@ if "env" in locals():
     # TODO: replace this with `env.Dump(format="json")`
     # once we start requiring SCons 4.0 as min version.
     methods.dump(env)
+
+
+def print_elapsed_time():
+    elapsed_time_sec = time.time() - time_at_start
+    time_ms = round((elapsed_time_sec % 1) * 1000)
+    print(f"[Time elapsed: {time.strftime('%H:%M:%S', time.gmtime(elapsed_time_sec))}.{time_ms:03}]")
```

Comment:
```py
>>> elapsed_time_sec = 5.9999
>>> time_ms = round((elapsed_time_sec % 1)*1000)
>>> print(f"[Time elapsed: {time.strftime('%H:%M:%S', time.gmtime(elapsed_time_sec))}.{time_ms}]")
[Time elapsed: 00:00:05.1000]
```

I think we need to round at the calculation of `elapsed_time_sec`

Reply:
Could truncate instead of rounding the ms and it should be fine.

Reply:
Now 5.9999 will result in [Time elapsed: 00:00:06.000]

Reply:
*sorry I accidently wrote wow instead of now :)